### PR TITLE
Multifits merge to release-0.2.0 after taking care of precommit changes

### PR DESCRIPTION
**Pull request by mpound**

### CODE REVIEW

**Comment by mpound** [comment]
There are still some files that are conflicted but far fewer than before.

File: .github/workflows/pre-commit.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  pre-commit:
+    env:
+      SKIP: ruff
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-python@v4
+      with:
+        # must match the version in .pre-commit-config.yaml:default_language_version.python
+        python-version: '3.9'
```
Comment (tchamberlin) [inline .github/workflows/pre-commit.yml:19]: Doesn't match the version in `.pre-commit.config.yaml` (3.10)

File: .github/workflows/hatch-and-pytest.yml [hunk 2]
```diff
@@ -5,14 +5,14 @@ name: Build and Pytest
 
 on:
   push:
-    branches: [ "main", "release-0.2.0", "cat-devel", "mwp-devel", "pedro-devel" ]
+    branches: [ "main", "release-0.1.0", "cat-devel", "mwp-devel", "pedro-devel", "evan-devel" ]
   pull_request:
-    branches: [ "main", "release-0.2.0", "cat-devel", "mwp-devel", "pedro-devel" ]
+    branches: [ "main", "release-0.1.0", "cat-devel", "mwp-devel", "pedro-devel", "evan-devel" ]
```
Comment (tchamberlin) [inline .github/workflows/hatch-and-pytest.yml:10]: Prefer wildcards here, e.g. `"main*"` and `"release-*"`; would also resolve the conflict

File: gui/app.py
Comment (tchamberlin) [inline gui/app.py:1]: Lots of changes here I'm not really in a position to analyze. Do we need to resolve conflicts or can we just keep the version in `multifits`?

File: attic/sdfitsload.py
Comment (tchamberlin) [inline attic/sdfitsload.py:1]: Lots of changes here I'm not really in a position to analyze. Do we need to resolve conflicts or can we just keep the version in `multifits`?

File: benchmark/makeplots.py
Comment (tchamberlin) [inline benchmark/makeplots.py:1]: I'm not really sure what to do here. Black clearly has not been run on the files in whatever version of `release-0.2.0` this is referencing. But I can see that the current `release-0.2.0` is formatted as expected

**Review by tchamberlin — CHANGES_REQUESTED** [review]
Bailing out of this review; we need to sort out the black formatting issues

**Comment by mpound** [comment]
ok let me work on this some more. 
@vcatlett you should address Thomas's question about gui/app.py

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -26,13 +24,16 @@ repos:
     rev: 5.12.0
     hooks:
       - id: isort
+        args: ["--profile", "black", "--filter-files"]
+        exclude: ^notebooks/
```
[the remark this answers is not in the document]
Comment (tchamberlin) [reply .pre-commit-config.yaml:28]: Sorry, this is confusing -- as per email, this is referencing only line 27; 28 is fine

**Pull request closed without merging**